src/pages/Cart/CartItems.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import './CartItems.scss';
+
+export const CartItems = ({ lists }) => {
+  return (
+    <div className="CartItems">
+      <ul className="listContainer">
+        {lists.map(function (item) {
```

Comment:
```suggestion
        {lists.map(function (item) {
```
매개변수 item에 어떤 데이터가 오는지 예상하기 쉽게 이름 짓는게 좋습니다